pkg/tfvars/baremetal/baremetal.go

```diff
@@ -75,13 +75,13 @@ func TFVars(libvirtURI, bootstrapProvisioningIP, bootstrapOSImage, externalBridg
 			"power_interface":      accessDetails.PowerInterface(),
 			"raid_interface":       accessDetails.RAIDInterface(),
 			"vendor_interface":     accessDetails.VendorInterface(),
+			"instance_info":        map[string]interface{}{},
```

Comment:
This should just get removed entirely. Having a map in here is throwing terraform off